From: Hanjun Guo <guohanjun@huawei.com>
Date: Tue, 24 Dec 2019 09:56:30 +0800
Subject: cpufreq: CPPC: put ACPI table after using it

Git-commit: 80e8b1e59f0399b94a6088bcb9477bd798cc5eba
Patch-mainline: v5.6-rc1
References: jsc#SLE-14771

Put the ACPI table to release the table mapping after using it
successfully.

Signed-off-by: Hanjun Guo <guohanjun@huawei.com>
[ rjw: Subject & changelog ]
Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>
Signed-off-by: Mian Yousaf Kaukab <yousaf.kaukab@suse.com>
---
 drivers/cpufreq/cppc_cpufreq.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/cpufreq/cppc_cpufreq.c b/drivers/cpufreq/cppc_cpufreq.c
index d0ca3007b4f2..a06777c35fc0 100644
--- a/drivers/cpufreq/cppc_cpufreq.c
+++ b/drivers/cpufreq/cppc_cpufreq.c
@@ -98,6 +98,8 @@ static void cppc_check_hisi_workaround(void)
 			break;
 		}
 	}
+
+	acpi_put_table(tbl);
 }
 
 /* Callback function used to retrieve the max frequency from DMI */
-- 
2.26.2

